### ISSUE TITLE
Property "preferredStatusBarStyle" of "UIViewController" has become readonly, leading to crash.

### ISSUE THREAD

**Issue by Yannmm** [cezarywojcik/CWStatusBarNotification#100]
`CWStatusBarNotification.m` line 381:

```
rootViewController.preferredStatusBarStyle = self.preferredStatusBarStyle;
```

This line cause carsh in iOS10, probably because apple has change the property `preferredStatusBarStyle` to readonly, thus can't be set so.

**Comment by protspace** [comment]
the same for me

**Comment by dheerajkatarya** [comment]
Same for me

**Comment by protspace** [comment]
added manually as a folder and it worked for me. Strange..

**Comment by Yannmm** [comment]
@protspace  you mean adding `CWStatusBarNotification` manually instead of `cocoapods`? you probably wanna `clean` your project and run it again, to make sure it works.😂

**Comment by protspace** [comment]
Yep, manually. Cleaning didn't work ) Probably Xcode 8 problem and i should delete Derived Data manually..

**Comment by Dewinster** [comment]
Renaming the preferredStatusBarStyle variable worked for me
